### ISSUE TITLE
hw3: may I assume that the user is not logged_in at the first time of rendering?

### ISSUE THREAD

**Issue by tippy33** [swsnu/swppfall2021#77]
Hello, while I was implementing logging function, I had a question regarding the logged status of the user.
Only at the very first time of logging at `/login` , can I assume the user("swpp@snu.ac.kr") is not logged_in, and after I put email and password correctly, the user's status will be changed into logged_in?
This was because if it is not assumed like this, I think a user who has logged in in the past, should be automatically moved to `/articles` , and this somehow gives me an infinite loop.. 
Or should I check every time whether the user is logged in even in the `/login`?

**Comment by kdh0102** [comment]
> Only at the very first time of logging at /login , can I assume the user("swpp@snu.ac.kr") is not logged_in, and after I put email and password correctly, the user's status will be changed into logged_in?

Yes. The first status of `logged_in` will be false, and when the user logs in successfully the status changes to true.
